galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.ipnetwork.manager/src/main/java/dev/galasa/ipnetwork/spi/SSHClient.java

```diff
@@ -254,7 +250,6 @@ public String issueCommandToShell(String command, boolean newShell) throws SSHEx
      *                 before timing out
      * @return the output of the command
      * @throws SSHException
-     * @throws JSchException
      */
     @Override
     public synchronized String issueCommandToShell(String command, boolean newShell, long timeout) throws SSHException {
```

Comment:
Might be more helpful to call the last parameter timeoutSeconds ? Then the user knows the units of the timeout also ?

Reply:
good idea!